### DISCUSSION TITLE
Email Clients

### DISCUSSION THREAD

**Discussion by ukwuezeobinna** [modoboa/modoboa#2175] [General]
I have tried everything and still can't get Modoboa to work with Thunderbird Mail.

Modoboa is installed using the install script default settings haven't changed.

Does anyone know how to fix this?

**Comment by MrGeneration** [comment]
It works absolutely fine with Outlook 365 and current Thunderbird.
You may have to provide login data manually through.

It also highly depends on your configuration, so e.g. error messages could be extremely useful.

**Comment by Sysadminfromhell** [comment]
On me its working in Outlook 365, Apple Mail and Microsoft Mail (AppX from Store). Whats your current error message?
Did you configure all neccessary DNS Entrys?

**Comment by ukwuezeobinna** [comment]
![Screenshot from 2021-02-12 16-19-02](https://user-images.githubusercontent.com/852830/107786735-86b7c800-6d4e-11eb-873c-d9122735a3e5.png)
![Screenshot from 2021-02-12 16-17-59](https://user-images.githubusercontent.com/852830/107787573-8d930a80-6d4f-11eb-8c12-d0cc62c3b45b.png)

Now i'm getting this.

**Reply by MrGeneration** [reply]
Ensure that your server is listening on Port 587 (SMTPs) and 993 (IMAPs).
If it does, ensure your firewall does not block these ports.

For this to work the server in question needs to use ssl certificates.

**Reply by ukwuezeobinna** [reply]
I used the install script and Lets Encrypt was installed.

Is there a way to re-issue the SSL Certificate?
